### PR TITLE
build multiarch images on amd64 machine

### PR DESCRIPTION
**Pull request by chenzhiwei**
fixes https://github.com/IBM/meta-operator/issues/39

The newly added targets will be skipped when running on non-amd64 machine.

/cc @morvencao @gyliu513 @DanielXLee @horis233

### CODE REVIEW

**Comment by gyliu513** [comment]
Fixed a makefile in https://github.com/IBM/meta-operator/pull/41

File: Makefile [hunk 1]
```diff
@@ -110,11 +111,46 @@ code-dev: ## Run the default dev commands which are the go tidy, fmt, vet then e
...
+	@strip build/_output/bin/$(IMG)
+
+build-image: build $(CONFIG_DOCKER_TARGET)
+	$(eval ARCH := $(shell uname -m|sed 's/x86_64/amd64/'))
+	docker build -t $(REGISTRY)/$(IMG)-$(ARCH):$(VERSION) -f build/Dockerfile .
+	@\rm -f build/_output/bin/meta-operator
+	@if [ $(BUILD_LOCALLY) -ne 1 ] && [ "$(ARCH)" = "amd64" ]; then docker push $(REGISTRY)/$(IMG)-$(ARCH):$(VERSION); fi
+
+# runs on amd64 machine
+build-image-ppc64le: build-ppc64le $(CONFIG_DOCKER_TARGET)
```
Comment (chenzhiwei) [inline Makefile]: when you copy these code, please remove the target `build-ppc64le`.

File: Makefile [hunk 2]
```diff
@@ -110,11 +111,46 @@ code-dev: ## Run the default dev commands which are the go tidy, fmt, vet then e
...
+	GOOS=linux GOARCH=ppc64le CGO_ENABLED=0 go build -o build/_output/bin/meta-operator-ppc64le ./cmd/manager
+	docker run --rm --privileged multiarch/qemu-user-static:register --reset
+	docker build -t $(REGISTRY)/$(IMG)-ppc64le:$(VERSION) -f build/Dockerfile.ppc64le .
+	@\rm -f build/_output/bin/meta-operator-ppc64le
+	@if [ $(BUILD_LOCALLY) -ne 1 ]; then docker push $(REGISTRY)/$(IMG)-ppc64le:$(VERSION); fi
+endif
+endif
+
+# runs on amd64 machine
+build-image-s390x: build-s390x $(CONFIG_DOCKER_TARGET)
```
Comment (chenzhiwei) [inline Makefile]: when you copy these code, please remove the target `build-s390x`.

**Comment by chenzhiwei** [comment]
template repo: https://github.com/IBM/go-repo-template/blob/master/Makefile#L139-L177

**Pull request merged**